x-pack/plugin/inference/src/test/java/org/elasticsearch/xpack/inference/services/azureopenai/AzureOpenAiSecretSettingsTests.java

```diff
@@ -143,6 +144,11 @@ protected AzureOpenAiSecretSettings mutateInstance(AzureOpenAiSecretSettings ins
         return randomValueOtherThan(instance, AzureOpenAiSecretSettingsTests::createRandom);
     }
 
+    @Override
+    protected AzureOpenAiSecretSettings mutateInstanceForVersion(AzureOpenAiSecretSettings instance, TransportVersion version) {
+        return instance;
```

Comment:
is the idea here that we only have one version, so there is no mutation to do?

Reply:
Yes exactly, it kinda acts as a reminder to implement mutation tests, if something on the transport level changes